compose/.apps/flaresolverr/flaresolverr.ports.yml

```
@@ -0,0 +1,4 @@
+services:
+  flaresolverr:
+    ports:
+      - ${FLARESOLVERR_PORT_8191}:8191
```

```suggestion
      - ${FLARESOLVERR_PORT_8191}:${FLARESOLVERR_PORT_8191}
```
Realized there's an environment variable to tell the app to run on a different port, so this is also needed.